packages/mds-compliance-service/repository/index.ts

```diff
@@ -132,10 +132,10 @@ export const ComplianceRepository = ReadWriteRepository.Create(
             .createQueryBuilder()
             .where('compliance_as_of >= :start_time', { start_time })
             .andWhere('compliance_as_of <= :end_time', { end_time })
-          if (isDefined(provider_ids)) {
+          if (isDefined(provider_ids) && provider_ids.length > 0) {
```

Comment:
One could make the argument that `= ANY([])` should return 0 rows by definition.

Reply:
What does the code calling us look like that is sometimes doing this?